Current Releases/head/roles.md

```diff
@@ -59,24 +59,68 @@ Often, security architects have to describe how their security solution impacts
+
+A senior developer, similar to an architect, should be able to explain to pen-testers and Security officers, what the architecture of the application landscape looks like, as well as the software development environment and platform.
+
+Senior developers should also have a firm knowledge of security controls like cryptography, identity and access management, secrets management and configuration, ssl and certificates. A senior developer should be able to explain how to implement these controls to medior developers and explain to pen-testers how these controls are implemented, which choices were made and why. Relevant to the project, a senior developer should be able to produce reusable components that implement security-related functionality and verify the quality of these components.
+
+For this quality verification, Senior developers should be able to implement and use quality assurance tools, as well as SAST and DAST tools, CI/CD tools and automation. The senior developer should be able to explain and help medior and junior developers to use these tools. Senior developers should also act as the first point of contact between the development team and pen-testing team. They should be able to translate pen-test results (vulnerabilities) to mitigations using security controls.
+
+### Manager
 
 
+
+### Product Owner
+A product owner should understand basic security principles, like the CIA triad. Therefore, a product owner should have followed standard security awareness training. He or she should have a firm understanding of the importance of developing secure software. A product owner should be able to translate non-functional requirements like quality in general, and more specific, performance, availability and security, to business value and / or risks. A product owner should be able to correctly prioritize the implementation of security controls, supported by senior developers, architects, security champions, pen-testers and security officers and be able to explain why these priorities are chosen to stakeholders.
+
 ### Security Champion
+A security champion is similar to a senior developer, in that he or she has a firm knowledge of security principles and is able to explain and help medior and junior developers in implementing security controls and use security frameworks and tools. 
 
+A Security Champion should be familiar with security additions to, and support the process of secure software development, including the use of OWASP resources like the OWASP Top 10's, OWASP Cheat-sheets, OWASP Tools like OWASP Dependency check, OWASP ZAP etc, and supporting documents and processes like OWASP SAMM and OWASP ASVS.
```

Comment:
Spelling; _A Security Champion should be familiar with security additions to,_ -> too

Reply:
I meant "security additions to the process of SSD" instead of "also"